backend/src/main/resources/application-okta-dev.yaml

```diff
@@ -3,6 +3,11 @@
 okta:
   oauth2:
     client-id: 0oa61vdmqd56921Ka4h6
-simple-report.authorization:
+simple-report:
+  authorization:
     role-claim: dev_roles
     role-prefix: "SR-DEV-TENANT:"
+  admin-emails:
+    - bwarfield@cdc.gov
+    - tim.best@usds.dhs.gov
+    - qom2@cdc.gov
```

Comment:
does `abeckett@cdc.gov` also work? might be nice to have some context of who `qom2` is

Reply:
Nope. In the authentication claims we have the principal email address as the username, ￼ and nothing else. This is temporary anyway, so I feel like the most we can do is put a comment here (if it were not, we could ask her to change her Okta login). ￼